### PR TITLE
Move model iteration's setvalues into mug

### PR DESCRIPTION
**Pull request by emord**
@millerdev Uses the correct api for setvalues, and no longer puts them in the form

buddy @czue

### CODE REVIEW

**Comment by emord** [comment]
@amstone326 This should fix the model iteration setvalue issue you were running into

**Comment by amstone326** [comment]
😄 😄

File: src/modeliteration.js [hunk 1]
```diff
@@ -86,7 +86,6 @@ define([
             },
             getExtraDataAttributes: function (mug) {
                 // HACK must happen before <setvalue> and "other" <instance> elements are written
-                prepareForWrite(mug);
```
Comment (millerdev) [inline src/modeliteration.js]: Does the `HACK` comment above this line pertain to this line only or to the whole method? Seems like the former, in which case it should also be removed.

Comment (emord) [inline src/modeliteration.js]: oops. yeah, missed that

**Comment by millerdev** [comment]
nice!

👍 on ✅

**Pull request merged**